servers/rest-services/src/main/java/org/projectnessie/services/rest/RestTreeResource.java

```diff
@@ -133,6 +133,7 @@ public EntriesResponse getEntries(String refName, EntriesParams params)
             params.namespaceDepth(),
             params.filter(),
             params.pageToken(),
+            false,
```

Comment:
Would it make sense to forbid (exception) `GetEntriesBuilder.withContent(true)` in API _v1_ for the sake of clarify?